tools/sentencepiece_testing/create_deberta_v3_test_proto.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tools.sentencepiece_testing.utils import _save
+from tools.sentencepiece_testing.utils import _train_sentencepiece
+
+
+def main():
+    bytes_io = _train_sentencepiece(
```

Comment:
For all of these let's just consolidate to a test single proto for each model.

So use the one `user_defined_symbols="[MASK]"`, and update any test output appropriately.

Reply:
Done.